feature/gribi/ate_tests/base_hierarchical_nhg_update/base_hierarchical_nhg_update_test.go

```diff
@@ -70,9 +73,40 @@ const (
 
 	// load balancing precision, %. Defines expected +-% delta for ECMP flows.
 	// E.g. 48-52% with two equal-weighted NHs.
-	lbPrecision = 2
+	lbPrecision     = 2
+	nh1ID           = 1
+	nh2ID           = 2
+	nh3ID           = 3
+	nh10ID          = 10
+	nh20ID          = 20
+	nhg1ID          = 1
+	nhg10ID         = 10
+	nhg20ID         = 20
+	innerDst        = "198.18.0.0/18"
+	innerDstPfx     = "198.18.0.0"
+	dstPfxvrf1      = "198.18.64.0"
+	dstPfxvalue     = "198.18.196.0"
+	mask            = "32"
+	port2mac        = "00:1A:11:00:1A:B9"
+	port3mac        = "00:1A:11:00:1A:BA"
+	port4mac        = "00:1A:11:00:1A:BB"
+	vip1            = "198.18.196.1"
+	outerSrcIP      = "203.0.113.0"
+	fps             = 1000000 // traffic frames per second
+	switchovertime  = 250.0
```

Comment:
May i know the reference documents 250 ms requirement.

Reply:
Its based on the convergence we see.

Reply:
Looks like there is not reference document to for expected convergences time so I think we need to have timer as deviation variable so that each vendor can input the value.

Reply:
Ok. Added the convergence time in args file now as expected value during path change.